### PR TITLE
Updated elm.rb from 0.19.0 to 0.19.1

### PR DESCRIPTION
**Pull request by vikrantrathore**
Updated elm to latest stable release version with some dependency updates.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
cabal: Could not resolve dependencies:
[__0] trying: elm-0.19.1 (user goal)
[__1] next goal: base (dependency of elm)
[__1] rejecting: base-4.10.1.0/installed-4.1... (conflict: elm => base>=4.11
&& <5)
[__1] rejecting: base-4.12.0.0, base-4.11.1.0, base-4.11.0.0, base-4.10.1.0,
base-4.10.0.0, base-4.9.1.0, base-4.9.0.0, base-4.8.2.0, base-4.8.1.0,
base-4.8.0.0, base-4.7.0.2, base-4.7.0.1, base-4.7.0.0, base-4.6.0.1,
base-4.6.0.0, base-4.5.1.0, base-4.5.0.0, base-4.4.1.0, base-4.4.0.0,
base-4.3.1.0, base-4.3.0.0, base-4.2.0.2, base-4.2.0.1, base-4.2.0.0,
base-4.1.0.0, base-4.0.0.0, base-3.0.3.2, base-3.0.3.1 (constraint from
non-upgradeable package requires installed instance)
[__1] fail (backjumping, conflict set: base, elm)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, elm
Note: when using a sandbox, all packages are required to have consistent
dependencies. Try reinstalling/unregistering the offending packages or
recreating the sandbox.
```

**Comment by maxekman** [comment]
Would be nice to get this merged! 🙂 Any idea when this could be completed?

File: Formula/elm.rb [hunk 1]
```diff
@@ -39,18 +43,18 @@ def install
         "source-directories": [
                   "."
         ],
-        "elm-version": "0.19.0",
+        "elm-version": "0.19.0 <= v < 0.20.0",
```
Comment (Razzeee) [inline Formula/elm.rb]: This is different, from what the compiler would create via `elm init`. A version range would normally only be used on `type` of `package`, but this is an `application`. See https://github.com/elm/compiler/blob/master/docs/elm.json/package.md for further reading.

**Comment by SMillerDev** [comment]
Closing in favor of #47353

**Comment by chenrui333** [comment]
Closing it now :)

**Pull request closed without merging**